src/components/global/Navbar.jsx

```diff
@@ -1,8 +1,147 @@
-const Navbar = () => {
-  // any logic needed
-  // return (
-  // all the JSX (looks like HTML) to render
-  // )
+import {
+  AppBar,
+  Box,
+  Hidden,
+  IconButton,
+  List,
+  ListItem,
+  Popover,
+  Toolbar,
+  Typography
+} from '@mui/material';
+import MenuIcon from '@mui/icons-material/Menu';
+import { Brightness4, Brightness7 } from '@mui/icons-material';
+import { BrowserRouter as Route, Link } from 'react-router-dom';
+import { useState, useRef } from 'react';
+
+const logoBlobStyle = {
+  position: 'absolute',
+  bottom: 0,
+  left: -30,
+  right: 0,
+  top: -50,
+  height: '250px',
+  backgroundImage: { md: 'url(/assets/logoBlobNav.svg)', sm: 'none' },
```

Comment:
This doesn't match the image name. Solved by either renaming the image itself to `logoBlobNav` or this line to `logoBlobnav`

Reply:
Im not sure what you are referring to. It is connected to the intended svg on line 24. Any name change will break that.

Reply:
the icon and brand name both redirect home, It's pretty commonplace to do that and not leave a home link. It only appears in the dropdown because the brand name drops off.  No plan on putting a home tab, might even remove it from dropdown in the future.

Reply:
In my assets folder, the image is titled `logoBlobnav.svg`. It's also what shows when I look through the `Files changed` tabbed. One has a capitalized 'Nav` while the other doesn't.

Reply:
![Screenshot 2023-10-26 at 6 00 45 PM](https://github.com/codeforpdx/codepdx_website/assets/114430929/9191a306-540c-495f-a9cd-990c6695a0fe)
Got it, thats a glitch of some sort, its capitalized on my local. Ill fix it after it merges but sticking with camelcase Nav would start with 'N'.

Reply:
The button does an eclipse transition when clicked and wont harm anyone if they tap it and its effect will be obvious. That is the icon MUI offers for dark mode (only one I saw). for the sake of trying to get the site up I'm OK with using it as is.

Reply:
In that case this PR looks good enough to me